lib/geordi/gitpt.rb

```diff
@@ -76,15 +77,26 @@ def load_projects
 
     def applicable_stories
       if Util.testing?
+        puts "Using the stories filter: '#{stories_filter}'"
```

Comment:
Actually, it would be nice to notify the user about the owner filtering. Instead of putsing the filter here, could you always `note 'Filtering stories by owner "$owner"'` if stories are filtered?